provider/provider_test.go

```diff
@@ -34,6 +36,24 @@ func getEnvRegion(t *testing.T) string {
 	return envRegion
 }
 
+func execPulumi(t *testing.T, ptest *pulumitest.PulumiTest, workdir string, args ...string) {
+	ctx := context.Background()
+	var env []string
+	workspace := ptest.CurrentStack().Workspace()
+	for k, v := range workspace.GetEnvVars() {
+		env = append(env, fmt.Sprintf("%s=%s", k, v))
+	}
+	stdin := bytes.NewReader([]byte{})
+	var arguments []string
+	arguments = append(arguments, args...)
+	arguments = append(arguments, "-s", ptest.CurrentStack().Name())
+	s1, s2, code, err := workspace.PulumiCommand().Run(ctx, workdir, stdin, nil, nil, env, arguments...)
+	t.Logf("import stdout: %s", s1)
```

Comment:
not necessarily import at this point, right? some arbitrary pulumi command stdout